mgplvm/kernels/stationary.py

```diff
@@ -52,7 +52,7 @@ def __init__(self,
             _scale_sqr = torch.tensor(scale,
                                       dtype=torch.get_default_dtype()).square()
         elif Y is not None:
-            _scale_sqr = torch.tensor(np.mean(Y**2, axis=(0, -1)))
+            _scale_sqr = torch.tensor(1 * np.mean(Y**2, axis=(0, -1)))
```

Comment:
why is this necessary?

Reply:
it's not, I think I was just playing around with different initializations to see if/how much it mattered